src/docs/development/ui/navigation/url-strategies.md

```diff
@@ -26,50 +26,52 @@ or [PathUrlStrategy][].
   setup.
 {{site.alert.end}}
 
-The `setUrlStrategy` API can only be called on the web. The following
+The `setUrlStrategy()` API can only be called on the web. The following
```

Comment:
I don't think we typically refer to function names using `()` in our docs, but I might be wrong.

Reply:
[Google developer documentation style guide](https://developers.google.com/style/code-in-text#method-names) says "Put an empty pair of parentheses after a method name to indicate that it's a method."

Reply:
:D  We generally only do that when it's otherwise unclear. But saying that you call (or invoke) `blank` makes it pretty clear it's a method. I'm sorry for any confusion! Sometimes the guidelines change over time. But, yeah, that's a totally fair comment.  :)